Creating a new README.md in English version. (#168)
ocodista/trampar-de-casa: README-en.md

# Trampar de Casa 💻🏠

![GitHub last commit (branch)](https://img.shields.io/github/last-commit/ocodista/trampar-de-casa/main)
![GitHub contributors](https://img.shields.io/github/contributors/ocodista/trampar-de-casa)
![Website](https://img.shields.io/website?up_message=online&up_color=green&down_message=offline&down_color=red&url=https%3A%2F%2Fwww.trampardecasa.com.br%2F)
![GitHub license](https://img.shields.io/github/license/ocodista/trampar-de-casa)

![Static Badge](https://img.shields.io/badge/nextjs-white?style=for-the-badge&logo=nextdotjs&logoColor=white&labelColor=black&color=white)
![Static Badge](https://img.shields.io/badge/typescript-%2306B6D4?style=for-the-badge&logo=typescript&labelColor=white&color=%233178C6)
![Static Badge](https://img.shields.io/badge/vite-%23646CFF?style=for-the-badge&logo=vite&labelColor=black)
![Static Badge](https://img.shields.io/badge/tailwindcss-%2306B6D4?style=for-the-badge&logo=tailwindcss&labelColor=black&color=%2306B6D4)

[Versão em português](./README.md)

## About the project 🎯

> The Trampar de Casa is an open-source
> an initiative dedicated to connecting Brazilian developers to remote work opportunities, encouraging inclusion, diversity, and through geographical boundaries.

Reviewer: ```suggestion
The Trampar de Casa is an open-source initiative dedicated to connecting Brazilian developers to remote work opportunities, encouraging inclusion, diversity, and eradicating geographical boundaries.
```

> Weekly, we send one email with job opportunities and 100% remotes matching our subscribers' profiles.

Reviewer: ```suggestion
On Wednesdays, we send one email with 100% remote job opportunities. National (🇧🇷) or International (🌍).
```
